src/main/java/org/session10/homework/bankaccountmanagementsystem/BankAccount.java

```diff
@@ -0,0 +1,75 @@
+package org.session10.homework.bankaccountmanagementsystem;
+
+import java.util.UUID;
+
+public class BankAccount {
+
+    private String accountHolderName;
+    private final UUID accountNumber;
+    private double balance;
+
+    public BankAccount(String accountHolderName) {
+        this();
```

Comment:
By using this() method that refer the current object, you are making use uf DRY Principle (Don't Repeat Yourself), because you centralize the UUID generation logic in one place. If in the future, the way you generate `accountNumber` changes, you'll need to change it in one place. Good job.